src/app/user/api/createUser.js

```diff
@@ -69,6 +77,27 @@ const createUser = async (req, res) => {
       );
     }
 
+    const emailUsername = email.toUpperCase().split("@")[0];
```

Comment:
Same comment as above about the comment in profile that says WHY we're doing this extra checking